includes/User.php

```diff
@@ -485,7 +485,7 @@ public static function newFromConfirmationCode( $code ) {
 		$id = $dbr->selectField( 'user', 'user_id', array(
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "user_id" 18 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)